---
title: "&#39; Schließlich &#39; kann nicht angezeigt werden, außerhalb einer &#39; Wiederholen Sie den &#39; Anweisung"
ms.date: 07/20/2015
ms.prod: .net
ms.technology:
- devlang-visual-basic
ms.topic: article
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
caps.latest.revision: 
author: dotnet-bot
ms.author: dotnetcontent
ms.openlocfilehash: 259df1345b443d5023ebecd457c81167d0799fe0
ms.sourcegitcommit: 34ec7753acf76f90a0fa845235ef06663dc9e36e
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 12/21/2017
---
# <a name="39finally39-cannot-appear-outside-a-39try39-statement"></a>&#39; Schließlich &#39; kann nicht angezeigt werden, außerhalb einer &#39; Wiederholen Sie den &#39; Anweisung
`Finally` wird zum Beenden eines `Try...Catch...Finally` -Blocks verwendet, kann daher nur einmal am Ende des Blocks angegeben werden. Entweder ist `Finally`unnötig, oder die `Finally` -Anweisung befindet sich außerhalb des entsprechenden `Try` -Blocks.  
  
 **Fehler-ID:** BC30382  
  
## <a name="to-correct-this-error"></a>So beheben Sie diesen Fehler  
  
1.  Suchen und entfernen Sie die unnötige `Finally` Anweisung.  
  
2.  Verschieben Sie die `Finally` -Anweisung an die gewünschte Position im Code.  
  
## <a name="see-also"></a>Siehe auch  
 [Try...Catch...Finally-Anweisung](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 
